Training/Model/Source/CustomSelect.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+namespace Codifi\Training\Model\Source;
+
+/**
+ * Class CustomSelect
+ * @package Codifi\Training\Model\Source
+ */
+class CustomSelect extends \Magento\Eav\Model\Entity\Attribute\Source\AbstractSource
+{
+    /**
+     * @return array|array[]|null
+     */
+    public function getAllOptions()
```

Comment:
1. Add description to the DocBlock. Example:
```
    /**
     * Get all options
     *
     * @return array
     */
```
2. In return type leave only `array` type